.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
   - phing recall -Dversion=$VERSION -Ddb.database=drupal -Ddb.user=lightning -Ddb.password=lightning -Ddb.host=127.0.0.1
   - cd docroot
   # Execute database updates, if there are any.
-  - drush updatedb --yes
+  - drupal update:execute
```

Comment:
Drush works for the updates, as long as you clear the cache afterwards to resolve inconsistencies. Is it OK if I update that bullet so that it mentions an explicit cache clear after the updates are done?